site/public/css/sidebar.css

```diff
@@ -20,7 +20,7 @@ aside {
     }
 
     aside i  {
-        color: var(--text-black);
+        color: var(--text-black) !important;
```

Comment:
is there a reason you are changing the sidebar css in this pr?

Reply:
This was needed to make the ? in icon black, otherwise it was made blue for some reason.